modules/utility/binds/which-key/config.nix

```diff
@@ -4,6 +4,7 @@
   ...
 }: let
   inherit (lib) mkIf nvim;
```

Comment:
In the `restructure` branch, I'm making lib calls more explicit. Would make my life easier if this PR also converted to the new format.

```nix
inherit (lib.options) mkIf
inherit (lib.nvim.lua) toLuaObject
```